### PR TITLE
fix: NPE on server startup

### LINKED ISSUE #20060: [Bug]: NullPointerException: The mapper [com.appsmith.server.solutions.ce.PluginScheduledTaskCEImpl$$Lambda$2190/0x000000080178e448] returned a null value.
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

I'm executing `docker-compose up` and expect for the app to run, however, I got the following  error:
```
reactor.core.Exceptions$ErrorCallbackNotImplemented:
java.lang.NullPointerException: The mapper [com.appsmith.server.solutions.ce.PluginScheduledTaskCEImpl$$Lambda$2059/0x00000008016c1748] returned a null value.

Caused by: java.lang.NullPointerException:
The mapper [com.appsmith.server.solutions.ce.PluginScheduledTaskCEImpl$$Lambda$2059/0x00000008016c1748] returned a null value
```

### Steps To Reproduce

1. `curl -L https://bit.ly/3h08WqB -o $PWD/docker-compose.yml`
2. `docker-compose up`

### Public Sample App

_No response_

### Issue video log

_No response_

### Version

Self Hosted - latest (v1.9.4)

### PR DESCRIPTION
**Pull request by sharat87**
Fixes #20060.

### CODE REVIEW

File: app/server/appsmith-server/src/main/java/com/appsmith/server/solutions/ce/PluginScheduledTaskCEImpl.java [hunk 1]
```diff
@@ -40,7 +42,7 @@ public class PluginScheduledTaskCEImpl implements PluginScheduledTaskCE {
 
 
     // Number of milliseconds between the start of each scheduled calls to this method.
-    @Scheduled(initialDelay = 30 * 1000 /* 30 seconds */, fixedRate = 2 * 60 * 60 * 1000 /* two hours */)
+    @Scheduled(initialDelay = 3 * 1000 /* 30 seconds */, fixedRate = 2 * 60 * 60 * 1000 /* two hours */)
```
Comment (trishaanand) [inline app/server/appsmith-server/src/main/java/com/appsmith/server/solutions/ce/PluginScheduledTaskCEImpl.java:45]:
```suggestion
    @Scheduled(initialDelay = 3 * 1000 /* 3 seconds */, fixedRate = 2 * 60 * 60 * 1000 /* two hours */)
```

**Review by trishaanand — APPROVED** [review]

**Review by trishaanand — APPROVED** [review]

File: app/server/appsmith-server/src/main/java/com/appsmith/server/solutions/ce/PluginScheduledTaskCEImpl.java (hunk 1 above)
Comment (sharat87) [reply app/server/appsmith-server/src/main/java/com/appsmith/server/solutions/ce/PluginScheduledTaskCEImpl.java]: Hahahaha, I just fixed this and was trying to push, and saw an error saying I have to pull first. Freaked the hell out of me thinking git broke 😂

Comment (sharat87) [reply app/server/appsmith-server/src/main/java/com/appsmith/server/solutions/ce/PluginScheduledTaskCEImpl.java]: Oh also, this was not an intentional change, it was done for testing. I'm actually going to revert it to 30, that okay?

**Review by trishaanand — APPROVED** [review]

**Comment by trishaanand** [comment]
@sharat87 : Is it really an error that cloud services did not return any SAAS plugins?

**Comment by sharat87** [comment]
@trishaanand, I do believe so, but that error isn't present on release-cs, so I'm guessing it was taken care of. I still wanted an NPE protection here, so opened this up.

**Pull request merged**